src/agent/onefuzz-agent/src/tasks/fuzz/generator.rs

```diff
@@ -234,23 +233,23 @@ mod tests {
         .collect();
 
         let radamsa_path = env::var("ONEFUZZ_TEST_RADAMSA_LINUX")?;
-        let radamsa_as_path = Path::new(&radamsa_path);
+        let radamsa_as_path = std::path::Path::new(&radamsa_path);
         let radamsa_dir = radamsa_as_path.parent().unwrap();
 
         let config = Config {
             generator_exe: String::from("{tools_dir}/radamsa"),
             generator_options,
             readonly_inputs: vec![SyncedDir {
-                path: inputs.to_path_buf(),
-                url: None,
+                path: tempfile::tempdir().unwrap().path().into(),
+                url: BlobContainerUrl::parse(Url::from_directory_path(inputs).unwrap())?,
             }],
             crashes: SyncedDir {
-                path: crashes.to_path_buf(),
-                url: None,
+                path: tempfile::tempdir().unwrap().path().into(),
+                url: BlobContainerUrl::parse(Url::from_directory_path(crashes).unwrap())?,
             },
             tools: Some(SyncedDir {
-                path: radamsa_dir.to_path_buf(),
-                url: None,
+                path: tempfile::tempdir().unwrap().path().into(),
```

Comment:
Does this work?  It's my understanding that as soon as `tempdir()` goes out of scope, the underlying paths get deleted.